src/portal-components/DetailsPage.tsx

```diff
@@ -428,7 +428,12 @@ export default class DetailsPage extends React.Component<
     return split.map((splitString) => {
       let value = splitString.trim()
       let entityTitle = ''
-      let lockedFacet: LockedFacet = {}
+
+      // For explorer 2.0, construct an object to contain the locked facet name and facet value
+      const lockedFacet: LockedFacet = {
+        facet: columnName
+      }
+
```

Comment:
Unlike Psychencode or AD knowledge portals, NF has resolveSynid set equal to false, we need to set the locked facet to column name regardless there is a resolveSynid object.

Reply:
ok